config/config.go

```diff
@@ -332,7 +332,7 @@ type HardforkConfig struct {
 
 // FullHistoryConfig holds the configuration for the full history node
 type FullHistoryConfig struct {
-	EnableHistoryNode               bool
+	Enabled                         bool
```

Comment:
This is `Enabled`, but in `config.toml` is `Enable`.

Reply:
renamed.